package.json

```diff
@@ -60,6 +60,7 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
For more meaningful assertions in test and more meaningful errors when those assertions fail.